PubHub/PubHub.API/Controllers/AuthController.cs

```diff
@@ -50,8 +51,10 @@ public AuthController(ILogger<AuthController> logger, PubHubContext context, Use
         [ProducesResponseType(StatusCodes.Status422UnprocessableEntity, Type = typeof(ProblemDetails))]
         public async Task<IResult> RegisterUserAsync([FromBody] UserCreateModel userCreateModel, [FromHeader] string appId)
         {
-            if (!_whitelistService.TryVerifyApplicationAccess(appId, GetType().Name, out IResult? problem))
-                return problem;
+            if (!_accessService.AccessFor(User, appId)
+                .CheckWhitelistEndpoint(GetType().Name)
```

Comment:
I don't think a one-liner would do good. This would be easier to read and bring more consistency to the way it is written in all the controller methods. It is similar to what you would do, when setting multiple contraints on a property in EF Core 😊

Reply:
Disregard this. GItHub was acting up.